cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -68,7 +71,33 @@ def test_appliance_scsi_control_vmware():
         initialEstimate: 1/4h
     #TODO: yet to test this, once done, I will add steps. Test was not written by me originally.
     """
-    pass
+    url = ("{}/{}/{}/cfme-vsphere-paravirtual-{}-1.x86_64.vsphere.ova"
+        .format(conf.cfme_data['basic_info']['cfme_images_url']['baseurl'],
+            appliance.version.series(), appliance.version, appliance.version))
+    logger.info("Downloading ova file for parvirtual vsphere scsi controller test from %s" % url)
+    ova_file = urllib.urlopen(url)
+    filename = "cfme-vsphere-paravirtual-{}.ova".format(appliance.version)
+    with open(filename, 'wb') as output:
+        output.write(ova_file.read())
+
+    @request.addfinalizer
+    def _cleanup():
+        if os.path.exists(filename):
+            os.remove(filename)
+    tar = tarfile.open(filename)
+    for member in tar.getmembers():
+        if member.name == 'desc.ovf':
```

Comment:
I think you are looking for one `member` in `tarfile`. What you think, if we just pick that member rather than iteration over `members`
```
desc_member = tar.getmember("desc.ovf")
```